app/assets/javascripts/vocab_bar.js

```diff
@@ -1,19 +1,45 @@
 $(document).ready(function() {
-  function hideVocabSheetOnMobile() {
-    var bar = $('.vocab_sheet_bar');
-    var windowWidth = $(window).width();
-    if (windowWidth < 640 || (bar.length && bar.find('.vocab_sheet_bar_item').length === 0)) {
-      bar.hide();
-      $('.not_sticky_footer').removeClass('vocab_sheet_background');
-    } else if (windowWidth > 639 &&  bar.length) {
-      bar.show();
-      $('.not_sticky_footer').addClass('vocab_sheet_background');
+  if ($('.vocab-sheet__bar').length > 0) {
+    var bar = $('.vocab-sheet__bar');
+
+    function hideVocabSheetOnMobile() {
+      var windowWidth = $(window).width();
```

Comment:
can this be achieved with a css media query? Or is there some voodoo magic in here somewhere that means it needs to be done with Javascript

Reply:
This was actually in there before, but it could be easily taken out for CSS so I'll just do that anyway.